procbuild/server.py

```diff
@@ -197,6 +197,8 @@ def _build_worker(nr):
 
     def build_and_log(*args, **kwargs):
```

Comment:
`*args, **kwargs` is such an impenetrable syntax.  It would be helpful to document either this method of `BuildManager.build_paper`.

Reply:
Agreed, I actually think this is an unnecessary framework that will go away once we're switching to zmq for messaging.

Reply:
I apologise if I was unclear, I was suggesting that we do not change this syntax in this PR. Rather, we wait to change this syntax in a later PR (e.g., #29).